Dockerfile

```diff
@@ -145,5 +145,6 @@ RUN apt-get -y update && \
         build-essential \
         cmake \
         ninja-build \
-        python-minimal \
-    && rm -rf /var/lib/apt/lists/*
+        python2.7 \
```

Comment:
just install python2, no need for links then.

Reply:
unfortunatelly there is no python2 package :(

Reply:
@darklukee any hint how to do it properly ?

Reply:
ok, I see. python2 package is available on ubuntu 20.04, not 18.04 which is used here.
You could use update-alternatives tool, but I'm fine with link.

Reply:
lets keep the link and fix in #16